warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/handler/edge/ProtobufEdgeDataTypeHandler.java

```diff
@@ -983,28 +623,6 @@ private NormalizedContentInterface getNullKeyedNCI(String fieldValue, Multimap<S
         return null;
```

Comment:
`getNullKeyedNCI` in PEDTHandler.java is now unused and can be deleted.

Reply:
I put this and a few other suggested removals in https://github.com/NationalSecurityAgency/datawave/pull/2269/files